src/data/lovelace.ts

```diff
@@ -91,6 +106,7 @@ export interface LovelaceBadgeConfig {
 export interface LovelaceCardConfig {
   index?: number;
   view_index?: number;
+  layout?: any;
```

Comment:
This should be removed right?

Reply:
This is for the layout config on each card. This can be used by the Custom Devs to set the layout on the cards and use that data to position them